server/src/test/java/org/elasticsearch/index/mapper/vectors/MultiDenseVectorScriptDocValuesTests.java

```diff
@@ -18,46 +18,48 @@
 import org.elasticsearch.script.field.vectors.MultiDenseVector;
 import org.elasticsearch.script.field.vectors.MultiDenseVectorDocValuesField;
 import org.elasticsearch.test.ESTestCase;
-import org.elasticsearch.test.index.IndexVersionUtils;
+import org.junit.BeforeClass;
 
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
 import java.util.Iterator;
-import java.util.List;
 
 import static org.hamcrest.Matchers.containsString;
 
 public class MultiDenseVectorScriptDocValuesTests extends ESTestCase {
 
+    @BeforeClass
+    public static void setup() {
+        assumeTrue("Requires multi-dense vector support", MultiDenseVectorFieldMapper.FEATURE_FLAG.isEnabled());
```

Comment:
What happens when we remove this feature flag? Should we rather reply on more stable `SearchCapabilities`?

Reply:
@mayya-sharipova feature flag is there as the initial work is all done over snapshot builds or specifically set behind a feature flag. This is OK here and if we ever remove the flag, that means its on and these setups can be removed. 

The capability checks are only for yaml test skipping to my knowledge.